style.css

```diff
@@ -0,0 +1,104 @@
+body {
+    /* display: flex; */
```

Comment:
you should not keep any commented code, pls remove all such occurences

Reply:
Okay